openapi/doc/index.rst

```diff
@@ -21,43 +21,49 @@ Authentication
 
 * Open menu ``[[ Settings ]] >> Users >> Users``
 * Select a user that will be used for iteracting over API
-* In ``REST API`` tab Click  ``[Generate Access Token]``
+* Next to ``REST API`` field, click  ``[Generate Access Token]``
+* Next to ``REST API`` field, specify token expiry
 * Copy **Basic Authentication code** to use in any system that support REST API (Open API)
-
+* If you want to refresh the token progrmatically from the connected system, also copy the **Refresh Token**.
```

Comment:
Do we really need refresh token? You specify Expiration date whatever you need (including not expire). Refresh token doesn't seem to make system more secury

Reply:
The refresh token is mainly so that the consumer API can script the auto renewal of the auth token. Imagine you have a long standing integration, and all of a sudden you need a manual intervention, let's say to keep syncing payments... This can be quite catastrophic.

Reply:
Why just don't implement one can that would be self-refreshing (as a workaround for 1 week session expiration) until you delete it from ``res.users``

Reply:
I got the idea about security. But let it be optional for one who doesn't want to make a script to refresh token.

Additionally, for security, we can make some logs of API requests (Time, API). Or least only last request